src/Command/GenerateKeysCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Helis\SettingsManagerBundle\Command;
+
+
+use ParagonIE\Paseto\Protocol\Version2;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class GenerateKeysCommand extends Command
+{
+    protected static $defaultName = 'helis:settings:generate-keys';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('helis:debug:settings:generate-keys')
```

Comment:
no need for this line

Reply:
also, use this command name instead of the one in `$defaultName`